Server/mods/deathmatch/logic/CResource.cpp

```diff
@@ -630,6 +636,14 @@ bool CResource::HasResourceChanged()
         }
     }
 
+    for (const auto& [strGlob, uiFileCount] : m_ResourceFilesCountPerDir)
+    {
+        std::vector<std::filesystem::path> files = glob::rglob(strDirPath.data() + strGlob);
```

Comment:
It's better to use `.c_str()` instead of `.data()` because these methods can do different things depends on C++ standard.

Reply:
Can't use this method on a string_view.

Reply:
Oh, okay. `str` prefix mislead me.